rkt/images.go

```diff
@@ -342,8 +345,35 @@ func (f *fetcher) downloadSignatureFile(sigurl string, out writeSyncer) error {
 
 // downloadHTTP retrieves url, creating a temp file using getTempFile
 // file:// http:// and https:// urls supported
-func (f *fetcher) downloadHTTP(url, label string, out writeSyncer) error {
-	res, err := http.Get(url)
+func (f *fetcher) downloadHTTP(aUrl, label string, out writeSyncer) error {
+	req, err := http.NewRequest("GET", aUrl, nil)
+	if err != nil {
+		return err
+	}
+	u, err := url.Parse(aUrl)
```

Comment:
You can use req.URL rather than parsing it again, NewRequest will handle this failure for you